src/Illuminate/Validation/Rule.php

```diff
@@ -20,11 +20,12 @@ class Rule
      *
      * @param  callable|bool  $condition
      * @param  array|string  $rules
+     * @param  array|string  $default
      * @return \Illuminate\Validation\ConditionalRules
      */
-    public static function when($condition, $rules)
+    public static function when($condition, $rules, $default = [])
```

Comment:
This is a breaking change unfortunately.

Reply:
Few days too late !

I will try differently

Reply:
Just realized I did not change the right method...

@driesvints `Rule::when` signature method must remain the same ? What about `ConditionalRules::__construct` ?

Reply:
Ah well to be fair if it's a very recent released thing we could maybe do it this way. I'll let @taylorotwell decide.

Reply:
Should I revert https://github.com/laravel/framework/pull/38450/commits/1daa1b3b67b2b0cd38ad416357e86242b6404864 to to get initial changes only ?

Reply:
@bastien-phi feel free yeah. Sorry about that.

Reply:
No problems ! Thanks for the review ;-)